docs/Using-EthSigner/Using-EthSigner.md

```diff
@@ -15,6 +16,55 @@ by EthSigner.
 !!! note 
     EthSigner supports the JSON-RPC service over HTTP only. 
 
+# eea_sendTransaction 
+
+Creates and signs a [private transaction](https://docs.pantheon.pegasys.tech/en/stable/Privacy/Privacy-Overview/)
+using the [keystore account](../Using-EthSigner/Getting-Started.md#create-password-and-key-files). 
+
+EthSigner submits the signed transaction to Pantheon using [`eea_sendRawTransaction`](https://docs.pantheon.pegasys.tech/en/stable/Reference/Pantheon-API-Methods/#eea_sendrawtransaction). 
+
+**Parameters**
+
+Transaction object for private transactions: 
+
+| Key             | Type                | Required/Optional                  | Value                                                                                                                         |
+|-----------------|--:-:----------------|----------------------------------- |-------------------------------------------------------------------------------------------------------------------------------|
+| **from**        | Data, 20&nbsp;bytes | Required                           | Address of the sender. Must be the address of the keystore account.                                                                                                        |
+| **to**          | Data, 20&nbsp;bytes | Not required for contract creation | `null` for contract creation transaction. Contract address for contract invocation transactions.                                                           |
+| **gas**         | Quantity            | Optional                           | Gas provided by the sender. Default is `90000`.                                                                               |
+| **gasPrice**    | Quantity            | Optional                           | Gas price provided by the sender in Wei. Default is `0`.                                                                      |
+| **nonce**       | Quantity            | Optional                           | Number of transactions sent from the `from` account before this one.                                                                    |
+| **data**        | Quantity            | Optional                           | Compiled contract code or hash of the invoked method signature and encoded parameters.                                        |
+| **privateFrom** | Data, 20&nbsp;bytes | Required                           | Orion address of the sender                                                                                                         |
```

Comment:
Don't think there is any need for this to be Pantheon/Orion specific. Perhaps just mention this is the public key of the sender  for the private transaction and maybe just use Orion as an example?

Reply:
@jframe -> I need to make it clear that it's the address of the Transaction Manager (eg, Orion) and not the Ethereum client (eg, Pantheon). So the update would have to be: 

| **privateFrom** |  Transaction manager (eg, Orion) address of the sender                                                                                                        
| **privateFor**  |  | Transaction manager (eg, Orion) addresses of recipients             

This makes it harder to read.  The EthSigner docs have this note: 
_The Ethereum client used in this documentation is Pantheon but EthSigner can be used with any Ethereum client._

What do you think about adding a similar note to explain Orion could be any transaction manager?

Reply:
I think adding note like that would work well

Reply:
Added the following note: 
Pantheon uses a Transaction Manager to implement privacy. [Orion](http://docs.orion.pegasys.tech) is the Transaction Manager used in this documentation but EthSigner can be used with other Transaction Managers.